mplc/corruption.py

```diff
@@ -156,8 +157,15 @@ def apply(self):
             raise Exception('Missing the Partner to duplicate')
         self.generate_matrix()
         idx = self.corrupted_index
+        if self.duplicated_partner.y_train.ndim == 1:
+            self.duplicated_partner.y_train = to_categorical(self.duplicated_partner.y_train.reshape(-1, 1))
+            one_label = True
```

Comment:
What is the meaning of `one_label` ?

Reply:
When your label is binary, labels are one dimensional, and not one-ho-encoded. All the corruption methods work with one-hot-encoded labels, so if there is only one label (for instance titanic, survive), we one-hot-encode it, then store the fact that we change the labels into one_label, then corrupt, then convert the labels back to their original format.

Reply:
Ok makes sense. Could you add a comment explaining this ?

Reply:
I will in the next PR, switch local/global val/test sets, which will present changes in corruptions, if that's ok for you